supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalInitializationActor.scala

```diff
@@ -0,0 +1,73 @@
+package cromwell.backend.impl.local
+
+import akka.actor.Props
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.impl.local.LocalInitializationActor._
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.validation.exception.ValidationAggregatedException
+import cromwell.backend.validation.{ContinueOnReturnCode, ContinueOnReturnCodeSet}
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.core._
+import lenthall.exception.AggregatedException
+import wdl4s.{Call, WdlExpression}
+
+import scala.concurrent.Future
+import scalaz.Scalaz._
+
+object LocalInitializationActor {
+  val FailOnStderrDefaultValue = false
+  val ContinueOnRCDefaultValue = 0
+
+  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configurationDescriptor: BackendConfigurationDescriptor): Props =
+    Props(new LocalInitializationActor(workflowDescriptor, calls, configurationDescriptor))
+}
+
+class LocalInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
+                               override val calls: Seq[Call],
+                               override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+  /**
+    * Abort all initializations.
+    */
+  override def abortInitialization: Future[WorkflowAbortResponse] = ???
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  override def validate: Future[Unit] = {
+    //TODO: Expressions need to be resolved. EvalFunctions is needed. TBD in the PR.
+    Future {
```

Comment:
Yes, that is one of the topics I would like to discuss since this code may need to be executed in JobExecutionActor.
I'll wait until I have a clear idea on how to share objects or dependencies within InitializationActors and ExecutionActors.